JOSS/paper.md

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
Is this the required format for this Markdown file? The formatting is very weird and looks more like reStructuredText than Markdown. For instance, the title/tags/authors/etc. show up as a very long nested table. Is this on purpose?

Reply:
This formatting is correct for JOSS, the pdf compiles.